concept_map.puml

```diff
@@ -26,22 +26,29 @@ cloud "GitHub Actions" as GA
 cloud Buildkite
 
 rectangle "CI integrations" as CI
+rectangle "Dart Cloud Functions" as DartCloudFunctions
 
 together {
     node Firebase {
         cloud Hosting
         database Firestore
         cloud Authentication
+        cloud "Cloud Functions" as CloudFunctions
+        cloud Analytics
```

Comment:
What do you think if we reduce some "clouds" from the concept map? For example, by making CIs in the above and Sentry as `node` or something else.